todo.py

```diff
@@ -66,16 +66,20 @@ def main():
 
         elif choice == '3':
             try:
+                todo_list.list_tasks()
                 task_number = int(input("Enter task number to delete: "))
                 todo_list.delete_task(task_number)
+                
```

Comment:
I think a way you could improve this is listing the tasks after the editing or deletion methods as well to ensure the operation was done correctly!

Reply:
I disagree with this. As is there is no way to do multiple deletions or additions and owing to the fact that this program prints out a successful or unsuccessful modification/deletion there is no point where printing out the list a second time or printing it after modification would be useful